build.cake

```diff
@@ -0,0 +1,115 @@
+// Install tools.
+#tool "nuget:https://www.nuget.org/api/v2?package=GitVersion.CommandLine&version=3.6.2"
+
+///////////////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+///////////////////////////////////////////////////////////////////////////////
+
+var target          = Argument<string>("target", "Default");
+var configuration   = Argument<string>("configuration", "Release");
+
+// Load other scripts.
+#load "./build/parameters.cake"
```

Comment:
@shiftkey it certainly wasn't meant to be ignored. Perhaps the choice of folder name `build` was included in the default gitignore file that @onovotny created. I didn't check. I borrowed the folder structure from another repo that I worked on. The intention would certainly be for it to work straight away after doing the clone.